### ISSUE TITLE
Data Fetching (Redux?)

### ISSUE THREAD

**Issue by EW0824** [EW0824/jobGPT#14]
- Cache and data fetching. Decide whether to use Axios or Redux.

**Comment by Twhidur-Rahman-Abid** [comment]
Can I fetch data with redux(RTK Query)?

**Comment by EW0824** [comment]
Certainly. RTK Query is great and optimised but probably too much overhead for this small project.
